lib/sales_analyst.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'sales_engine'
+require'pry'
+
+class SalesAnalyst
+  attr_reader :se
+
+  def initialize(se)
+    @se = se
+  end
+
+  def average_items_per_merchant
```

Comment:
Ok, whatever you think is best given the other methods. I'll merge this for now but we can always modify.